tests/backend.cpp

```diff
@@ -167,30 +170,27 @@ TEST_F(backendTest, loadOne) {
 		std::chrono::system_clock::now() + deadlock_threshold;
 	while(argo::backend::atomic::load(_c) != c_const)
 		ASSERT_LT(std::chrono::system_clock::now(), max_time);
-	ASSERT_EQ(c_const, *_c);
```

Comment:
This is a slight change of behaviour, but ultimately I think it is sufficient to check that the while loop has finished in order to verify the removed `ASSERT_EQ`.